eng/Dependencies.props

```diff
@@ -82,12 +82,14 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.Extensions.WebEncoders" Version="$(MicrosoftExtensionsWebEncodersPackageVersion)" />
     <LatestPackageReference Include="Microsoft.JSInterop" Version="$(MicrosoftJSInteropPackageVersion)" />
     <LatestPackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryPackageVersion)" />
+    <LatestPackageReference Include="Microsoft.Win32.SystemEvents" Version="$(MicrosoftWin32SystemEventsPackageVersion)" />
```

Comment:
In servicing builds we hoist transitive references to be direct references in the .Ref and .Runtime projects. A few of the `@(_TransitiveExternalAspNetCoreAppReference)` items were missing here.